aries-site/src/pages/templates/global-banner-notifications.mdx

```diff
@@ -68,10 +72,10 @@ For low-severity notifications that do not require user action or user attention
 ### Format
 
 - Only show one banner at a time.
-- Global Banners should stay within the 192 character limit.
+- Global Notifications should stay within the 192 character limit.
 - Long text is not truncated but instead will wrap on smaller screen sizes.
-- Anchors are left-aligned and will wrap alongisde the text.
-- Use only two or fewer anchors.
+- Links are left-aligned and will wrap alongisde the text.
```

Comment:
The component is called "Anchor", can we align as a team— maybe via a thread in staff— what we are collectively calling this component in guidance, anchor or link?

If we have decided we're going with "link", I don't remember/understand the rationale. And vice versa if we are going with "anchor". Would be good to align as a group.

Reply:
I thought we said designers may not understand `Anchor` and `link` was more used but Lucas can verify

Reply:
This comes from a comment in the refactor Figma file. I proposed to unify in the word `link` in this case as it is more common among devs and designers. Also the anchor element is comprised of both the linked anchor and the target named anchor where the page scrolls to. This may be confusing about the expected behavior (navigating to another page vs scrolling down to the target anchor.

I think we should discuss this with the team next week.